templates/_base.html

```diff
@@ -5,6 +5,9 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
 <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+<meta property="og:image" content="/static/images/portmap_meta.jpg" />
+<meta property="twitter:card" content="/static/images/portmap_meta.jpg" />
```

Comment:
This one should be
```suggestion
<meta property="twitter:card" content="summary_large_card" />
```

https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image